queries/ada/textobjects.scm

```diff
@@ -0,0 +1,22 @@
+;; Support for high-level text objects selections.
```

Comment:
textobjects live in a separate repo github.com/nvim-treesitter/nvim-treesitter-textobjects

Reply:
ping

Reply:
Oups, I thought I had removed it previously, sorry